src/main/fabric8/configmap.yml

```diff
@@ -0,0 +1,9 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  creationTimestamp: 2016-02-18T19:14:38Z
+  name: app-config
+data:
+  app-config: |
+    message : "Hello, %s from a ConfigMap !"
+    level : INFO
```

Comment:
Adding the config map here is what we try to avoid as it means that the config is deployed at the same time as the application. So why do you care about using a config map in this case, and not use just a file in your classpath.